arm-web/2016-08-01/swagger/WebApps.json

```diff
@@ -13409,255 +13420,488 @@
     "SiteConfig": {
```

Comment:
ok, please check properties sharing described above can be applied for SiteConfigResource and SiteConfig